toolchain/check/import_ref.cpp

```diff
@@ -1253,11 +1257,23 @@ class ImportRefResolver {
       case CARBON_KIND(SemIR::UnboundElementType inst): {
         return TryResolveTypedInst(inst);
       }
-      default:
-        context_.TODO(
-            SemIR::LocId(AddImportIRInst(inst_id)),
-            llvm::formatv("TryResolveInst on {0}", untyped_inst.kind()).str());
-        return {.const_id = SemIR::ConstantId::Error};
+      default: {
```

Comment:
This change is necessary to support cases where the instruction that we are importing has a constant value, but is not the instruction that *defines* the constant value. For example, this happens when we have `extend base: B`, where the `B` is a `NameRef` instruction (which we won't directly import).